### PR TITLE
Update Jump Feeds

### PR DESCRIPTION
**Pull request by davevsdave**

### CODE REVIEW

**Review by bddq — APPROVED** [review]

**Review by jcn — CHANGES_REQUESTED** [review]
Thanks for this PR! Can you update the System Names to match what's listed in the feeds themselves (which will also help differentiate between the scooter and bike feeds). Thanks.

**Comment by davevsdave** [comment]
These changes should be addressed. Let me know if you need something additional.

File: systems.csv [hunk 1]
```diff
@@ -204,20 +204,25 @@ US,Indego,"Philadelphia, PA",bcycle_indego,https://www.rideindego.com,https://gb
...
-US,JUMP NYC,"NYC, NY",jump_nyc,https://jump.com/,https://nyc.jumpbikes.com/opendata/gbfs.json
-US,JUMP Providence,"Providence, RI",jump_providence,https://jump.com/,https://pvd.jumpbikes.com/opendata/gbfs.json
-US,JUMP Sacramento,"Sacramento, CA",jump_sacramento,https://jump.com/,https://sac.jumpbikes.com/opendata/gbfs.json
-US,JUMP San Diego,"San Diego, CA",jump_san_diego,https://jump.com,https://san.jumpbikes.com/opendata/gbfs.json
-US,JUMP SC,"Santa Cruz, CA",jump_sc,https://jump.com/,https://sc.jumpbikes.com/opendata/gbfs.json
-US,JUMP Seattle,"Seattle, WA",jump_seattle,https://jump.com,https://sea.jumpbikes.com/opendata/gbfs.json
-US,JUMP SF,"San Francisco, CA",jump_sf,https://jump.com/,https://sf.jumpbikes.com/opendata/gbfs.json
+US,Austin Bike JUMP System,"Austin, TX",austin_bike_jump_system,https://jump.com/,https://gbfs.uber.com/v1/atxb/gbfs.json
+US,Austin Scooter JUMP System,"Austin, TX",austin_scooter_jump_system,https://jump.com/,https://gbfs.uber.com/v1/atxs/gbfs.json
+US,Chicago Bike JUMP System,"Chicago, IL",jump_chicago,https://jump.com/,https://chi.jumpbikes.com/opendata/gbfs.json
```
Comment (jcn) [inline systems.csv]: This does not appear to be an active feed (all subpages 404).

**Review by jcn — CHANGES_REQUESTED** [review]
Thanks for the update. One last comment - looks like the Chicago Bike JUMP url is 404ing for all internal pages. Is there a better URL for that one?

File: systems.csv (hunk 1 above)
Comment (davevsdave) [reply systems.csv]: Yea, so Chicago revoked their temporary permit about a month ago for all scooters/bikes. I think they are going to renew in the spring. Up to you all if you want to remove and then add back in, happy to update the commit by removing that URL. LMK if that's what you'd like.

Comment (jcn) [reply systems.csv]: @davevsdave Sorry, let's remove it and keep the file full of valid feeds for now. Thanks for taking the time with these edits.

Comment (davevsdave) [reply systems.csv]: All good - The Chicago jump feeds have been removed and I also popped in the BOLT LA feed.

**Review by jcn — APPROVED** [review]

**Pull request merged**